src/main/java/uk/gov/hmcts/dts/fact/entity/Court.java

```diff
@@ -117,4 +117,8 @@
             .sorted(comparing(AreaOfLaw::getName))
             .collect(Collectors.toList());
     }
+
+    public boolean isInPerson() {
+        return inPerson == null || inPerson.getIsInPerson();
```

Comment:
Is it better to not refer to a null here? 

Also a comment may help in terms of understanding why if it is null, it means that it should be true

Reply:
I will add a comment here but we need a NULL check as the in-person table on database is incomplete. So if a court does not have any linked information in the in-person court table (i.e. the inPerson court object is NULL), it will be considered an in-person court. The front end currently depends on this logic. If we remove the NULL check, all the courts currently considered an in-person court will suddenly use the not in-person court template and the court page will change.